src/math/types/seconds_per_liquidity.rs

```diff
@@ -129,17 +131,15 @@ pub mod tests {
             let liquidity = Liquidity::new(1);
             let current_timestamp = 315360000;
             let last_timestamp = 0;
-            let seconds_per_liquidity =
-                SecondsPerLiquidity::calculate_seconds_per_liquidity_global(
-                    liquidity,
-                    current_timestamp,
-                    last_timestamp,
-                )
-                .unwrap();
-            assert_eq!(
-                seconds_per_liquidity.get(),
-                315360000000000000000000000000000000000
-            );
+            let (_, cause, stack) = SecondsPerLiquidity::calculate_seconds_per_liquidity_global(
+                liquidity,
+                current_timestamp,
+                last_timestamp,
+            )
+            .unwrap_err()
+            .get();
+            assert_eq!(cause, "conversion to contract::math::types::seconds_per_liquidity::SecondsPerLiquidity type failed");
```

Comment:
This case is inside the domain and has failed. This indicates that you have the wrong decimal selected for `SecondsPerLiquidity`. Start by updating the math specification